staging/src/k8s.io/kubectl/pkg/drain/drain_test.go

```diff
@@ -124,13 +141,29 @@ func TestDeletePods(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.description, func(t *testing.T) {
 			_, pods := createPods(false)
-			ctx := context.Background()
+			var ctx context.Context
+			var cancel context.CancelFunc
+			ctx = context.Background()
 			if test.ctxTimeoutEarly {
-				ctx, _ = context.WithTimeout(ctx, 100*time.Millisecond)
+				ctx, cancel = context.WithTimeout(ctx, 100*time.Millisecond)
```

Comment:
go vet complains if you don't defer this cancel even though we should never need to, thus the ugly var declarations above.